community.markdown

```diff
@@ -8,6 +8,10 @@ isCommunity: true
 
 Haskellers interact, talk and collaborate across several mediums and around the world. There are places to learn, to teach, to ask questions, and to find contributors and collaborators.
 
+## Supporting the Community
+
+If you would like to support the Haskell community, consider [donating to Haskell.org](../donations), the organization that maintains a significant part of the Haskell community infrastructure.
+
```

Comment:
I am not sure if this is the right place to link to this -- could we move this to below the list of communities?

Reply:
Good idea.